accounting/fiscal_localizations/localizations/chile.rst

```diff
@@ -0,0 +1,627 @@
+so all your billing users can use the same one. On the other hand,  if you need to restrict the
+certificate for a particular user, just define the users in the certificate. 
+
+Financial Reports
+~~~~~~~~~~~~~~~~~
+The report Propuesta F29 requires two values that need to be defined as part of the company
+configuration: 
+
+.. image:: media/Chile05.png
+   :align: center
+
+Multicurrency
+~~~~~~~~~~~~~
+The official currency rate in Chile is provided by the SBIF, you can find the SBIF
+synchronization service in the currency rate configuration and you can set a predefined interval
+for the rate updates.  
+
+.. image:: media/Chile05_2.png
+   :align: center
+   :scale: 25
+
+In order to configure your token, you need to enable developer mode and access to
+:menuselection:`Settings --> Technical --> System parameters` and search the parameter
+``sbif_api_key`` and in the field Value introduce your token provided by the SBIF:
+
+.. image:: media/Chile05_3.png
+   :align: center
+   
+Chart of accounts
+~~~~~~~~~~~~~~~~~
+The chart of accounts is installed by default as part of the set of data included in
+the localization module, the accounts are mapped automatically in: 
+
+- Taxes.
+- Default Account Payable.
+- Default Account Receivable.
+- Transfer Accounts.
+- Conversion Rate.
+
+
+Master Data
+-----------
+
+Partner
+~~~~~~~
+Identification Type and VAT
+***************************
+As part of the Chilean localization, the identification types defined by the SII
+are now available on the Partner form, this information is essential for most transactions.
+
+.. image:: media/Chile06.png
+   :align: center
+   :scale: 25
+ 
+TaxpayerType
+************
+In Chile the document type associated with customers and vendors transactions is defined
+based on the Taxpayer Type, this field should be defined in the partner form, when creating
+a customer is important you make sure this value is set:
+
+.. image:: media/Chile07.png
+   :align: center
+ 
+
+Electronic Invoice Data
+***********************
+As part of the information that is sent in the electronic Invoice, you need to define the
+email that is going to appear as the sender of the electronic invoice to your customer, and
+the Industry description. 
+
+.. image:: media/Chile07_2.png
+   :align: center
+ 
+
+Taxes
+~~~~~
+As part of the localization module the taxes are created automatically with their related
+financial account and configuration.
+
+.. image:: media/Chile08.png
+   :align: center
+
+Taxes Types
+***********
+Chile has several tax types, the most common ones are:
+
+- VAT. Is the regular VAT and it can have several rates. 
+- ILA (Impuesto a la Ley de Alcholes). Taxes for alcoholic drinks, it has a different rate.
+
+
+Fiscal Positions
+~~~~~~~~~~~~~~~~
+Based on the purchase transactions the VAT can have different affections, this will be done
+in Odoo using the default purchase fiscal positions. 
+
```

Comment:
```suggestion
Based on the purchase transactions, the VAT can have different affections. This will be done
in Odoo using the default purchase fiscal positions. 

```